src/glossary.md

```diff
@@ -59,18 +59,32 @@ and functions that evaluate to a single value, with or without side-effects.
 
 For example, `2 + (3 * 4)` is an expression that returns the value 14.
 
+### Free function
```

Comment:
This can be generalized to "free item", e.g. for a free const (not associated) or free static (we don't have associated statics). This should also be moved to after "associated item" and be worded *in opposition* to associated items.

Reply:
Unalphabetized glossaries are confusing.

Reply:
Yes fair enough. Maybe link back and forth instead?